src/hotspot/cpu/ppc/assembler_ppc.inline.hpp

```diff
@@ -130,6 +143,17 @@ inline void Assembler::divw_(  Register d, Register a, Register b) { emit_int32(
 inline void Assembler::divwu(  Register d, Register a, Register b) { emit_int32(DIVWU_OPCODE  | rt(d) | ra(a) | rb(b) | oe(0) | rc(0)); }
 inline void Assembler::divwu_( Register d, Register a, Register b) { emit_int32(DIVWU_OPCODE  | rt(d) | ra(a) | rb(b) | oe(0) | rc(1)); }
 
+// Prefixed instructions, introduced by POWER10
+inline void Assembler::paddi(  Register d, Register a, long si34, bool r = false) {
+  assert(a != R0 || r, "r0 not allowed, unless R is set (CIA relative)");
+  paddi_r0ok( d, a, si34, r);
```

Comment:
The space after the ( isn't needed here, since it's not aligning with a similar call above or below.